modules/gitops-argocd-rbac.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argocd-rbac_{context}"]
+= Argo CD role-based access control (RBAC)
+
+Access controls in {gitops-title} are managed at two distinct levels as follows:
+
+* At the Kubernetes level, the {gitops-shortname} Argo CD Application Controller interacts with one or more clusters to deploy various resources by using a single Kubernetes service account per cluster. This service account must have sufficient permissions to deploy resources for all tenants and use cases that this instance of Argo CD is managing.
+* At the Argo CD level, the {gitops-shortname} Argo CD Application Controller includes its own RBAC permissions model independent of Kubernetes. To manage the user level access, you can use this RBAC model independently of the underlying Kubernetes permissions. You can use this capability to provide access to Argo CD applications that a user would not have access to from a purely Kubernetes perspective.
+
+Because these two access controls and the interaction between components are distinct and separate, privilege escalation becomes a concern when designing a multitenant solution with Argo CD. Privilege escalation is when a tenant leverages the increased privileges of the application controller’s service account to perform actions they would not typically be permitted to perform. You can mitigate the privilege escalation in an Argo CD instance by using the Argo CD RBAC or separate instances of Argo CD.
```

Comment:
@gnunn1 Thanks for the information. Ack.

@Dhruv-Soni11 Please take note of this, create a JIRA, and mention these details in the description. It will be useful to track the work and complete it when this feature becomes available with doc inputs.  Kindly share the JIRA issue ID and link here. Also, please link this PR to the JIRA issue you create. Thank you!

Reply:
Hello @Srivaralakshmi, I have created the following JIRA ticket [RHDEVDOCS-6321](https://issues.redhat.com/browse/RHDEVDOCS-6321), which includes an updated description to address changes as per this comment or any other feedback that may arise once the alpha feature is ready for consumption. Kindly have a look and let me know if it is fine.

Reply:
Thanks, @Dhruv-Soni11, LGTM. About the Fix version, I am not sure whether it has to be done in 1.16. I leave that to you and Kavya to decide.